dev/docker/onedal-dev.Dockerfile

```diff
@@ -0,0 +1,59 @@
+#===============================================================================
+# Copyright 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#===============================================================================
+
+FROM ubuntu:22.04
```

Comment:
Probably unnecessary, but I constantly run into stuff associated with infra RHEL support vs ubuntu. Any idea on how general this docker container could be? (Looks like almost any recent debian based -nix version might be able to run it), would it be worth it doing a fedora one? (just an idea, doesn't need to be in this PR if yes)

Reply:
we might try adding it as well if this would work + would be extended to other components